RSignatureVerifier/Verifier.swift

```diff
@@ -0,0 +1,49 @@
+internal class Verifier {
```

Comment:
I didn't expect any functionality to be added to the project yet.
I thought that'd be in a separate PR but if it's included and the tests pass then I think it's OK.